content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -144,6 +144,14 @@ dependencies {
 
 {{% /tab %}} {{< /tabpane>}}
 
+#### Disable data export
+
+You can disable data export, e.g. for testing purposes.
+
+```properties
+otel.sdk.disabled=true
+```
```

Comment:
do we have a consistent way in other Java documentation of describing system properties / env vars?

Reply:
@trask great point - I've used the standard widget and also fixed all the other places